script/jenkins/linux-test.sh

```diff
@@ -11,16 +11,6 @@ jenkins_setup
 #}
 #trap cleanup EXIT
 
-make test-deps testsuite
-
-export COV_DIR="build/coverage"
-
-for i in "full.cov" "integration.cov" "system.cov" "unit.cov"
-do
-  name=$(basename ${i} .cov)
-  if [ -f "${COV_DIR}/${i}" ]; then 
-    go tool cover -html="${COV_DIR}/${i}" -o "${COV_DIR}/coverage-${name}-report.html"
-    gocover-cobertura < "${COV_DIR}/${i}" > "${COV_DIR}/coverage-${name}-report.xml"
-  fi
-done
-exit 0
+make update
+make system-tests-environment
+make stop-environment
```

Comment:
yes, that is intentional. the `integration` build just runs the unit tests in a provided docker env as far i can see. i think it is redundant since we have a separate unit test stage. if you think it is important, we could update that stage to use the docker env _instead_, but run them twice is a waste imo
i also don't see the point at all of dockerizing go unit tests

Reply:
I see your point with the docker env, and I agree that we should not run the same tests more often than necessary. 

My concerns refer to `go test -tags=integration` not being run anywhere anymore. Which, again is not an issue at the moment, we just have to ensure to add it in case we do make use of the golang [build constrain](https://golang.org/pkg/go/build/#hdr-Build_Constraints) for integration tests in the future. 

So these changes are fine with me, just wanted to be explicit about implications.